src/core/devutils/debughandlers/resizedebughandler/ResizeDebugHandler.ts

```diff
@@ -0,0 +1,37 @@
+import { DebugHandler } from "../DebugHandler";
+import ResizeDebugConfig from "./ResizeDebugConfig";
+import { Dimension } from "../../../..";
+import DebugConfig from "../DebugConfig";
+
+export default class ResizeDebugHandler implements DebugHandler {
+    private readonly _resizeDebugConfig: ResizeDebugConfig;
+
+    constructor(resizeDebugConfig: ResizeDebugConfig) {
+        this._resizeDebugConfig = resizeDebugConfig;
+    }
+
+    public resizeDebug = (oldDim: Dimension, newDim: Dimension, index: number): void => {
+        const expectedDim: Dimension = { width: 0, height: 0 };
```

Comment:
why do we want this ? Can't we share the old and newDim as it is ?

Reply:
I didn't want to share the dimension info in which client wasn't interested in.